### PR TITLE
Issue 56 integrate next and laravel

### PR DESCRIPTION
**Pull request by l443018**
#56

### CODE REVIEW

File: src/modules/auth/services/interfaces/LoginResponse.d.ts [hunk 1]
```diff
@@ -25,4 +25,11 @@ export interface LoginResponse {
    * @memberof LoginResponse
    */
   token_type: string;
+
+  /**
+   *
+   * @type {string|null}
+   * @memberof LoginResponse
+   */
+  email: string;
```
Comment (ericHao22) [inline src/modules/auth/services/interfaces/LoginResponse.d.ts]: email attribute seems to be not exist on `LoginResponse` interface

Comment (ericHao22) [reply src/modules/auth/services/interfaces/LoginResponse.d.ts]: Never mind. I see the fixup at next commit

**Comment by ericHao22** [comment]
r+, thanks!!

**Comment by koshuang** [comment]
r+, thanks!

**Pull request merged**